FSharp.AutoComplete/Program.fs

```diff
@@ -617,6 +641,36 @@ module internal Main =
 
               main state
 
+          | SymbolUse ->
+              let symboluses =
+                  async {
+                      let! symboluse = tyRes.GetSymbol(line, col, lineStr)
+                      if symboluse.IsNone then return None else
+                      let! symboluses = tyRes.GetUsesOfSymbolInFile symboluse.Value.Symbol
+                      return Some {
+                        Name = symboluse.Value.Symbol.DisplayName
+                        Uses =
+                          [ for su in symboluses do
+                              yield { StartLine = su.RangeAlternate.StartLine
+                                      StartColumn = su.RangeAlternate.StartColumn + 1
+                                      EndLine = su.RangeAlternate.EndLine
+                                      EndColumn = su.RangeAlternate.EndColumn + 1
```

Comment:
It is all very confusing. Adding 1 to `StartColumn` and `EndColumn` seems to be the right thing to do, so we normalize to consistently being 1-based . Do the other commands return 0-based?